controllers/storagecluster/clusterclaims.go

```diff
@@ -193,6 +203,26 @@ func (c *ClusterClaimCreator) getCephFsid() (string, error) {
 	return "", fmt.Errorf("failed to fetch ceph fsid from %q secret", RookCephMonSecretName)
 }
 
+func (c *ClusterClaimCreator) getIsDROptimized(serverVersion *version.Info) (string, error) {
+	var cephCluster rookCephv1.CephCluster
+	err := c.Client.Get(c.Context, types.NamespacedName{Name: generateNameForCephClusterFromString(c.StorageCluster.Name), Namespace: c.StorageCluster.Namespace}, &cephCluster)
+	if err != nil {
+		return "false", err
+	}
+	if cephCluster.Status.CephStorage == nil || cephCluster.Status.CephStorage.OSD.StoreType == nil {
+		return "false", fmt.Errorf("cephcluster status does not have OSD store information")
+	}
+	bluestorerdr, ok := cephCluster.Status.CephStorage.OSD.StoreType["bluestore-rdr"]
+	if !ok {
+		return "false", nil
+	}
+	total := getOsdCount(c.StorageCluster, serverVersion)
+	if bluestorerdr < total {
```

Comment:
Do we care if the cluster is still migrating? 
In that scenario, `bluestore-rdr` can be less that `total`

@umangachapagain

Reply:
I asked that same question to @GowthamShanmugam . We just want to know if migration is complete. Everything else is considered unsupported/false.